wherehows-web/app/templates/components/dataset-author.hbs

```diff
@@ -1,6 +1,15 @@
 <td>
   {{user-avatar userName=owner.userName}}
+
   {{owner.userName}}
+
+  {{#if isOwnerInActive}}
+
+    <button class="nacho-button nacho-button--small dataset-author-record__action--inactive">
```

Comment:
the `.dataset-author-record__action--inactive` class looks like it has a strikethrough decoration. Which means here you have the word "Inactive" with a strikethrough. Doesn't seem to make sense from an aesthetic standpoint to me. Shouldn't it be like the user or some other meta info that gets struck through rather than a button?

Reply:
In reality it actually doesn't have a strikethrough :) haha gotta up your matrix reading :trollface:

Reply:
the button text does not have a strikethrough, just the text in the cells